build_docs

```diff
@@ -234,7 +234,7 @@ def run_build_docs(args):
                 # is weird. If we tell it that the browser's name is `open`
                 # we'll get whatever the user set as their default browser.
                 environ['BROWSER'] = 'open'
-            webbrowser.open('http://localhost:8000', new=1, autoraise=False)
+            webbrowser.open('http://localhost:8000/guide', new=1, autoraise=False)
```

Comment:
This is the new path. It mirrors the public site nicely when you do `./build_docs --all --open`